setup.py

```diff
@@ -45,10 +45,10 @@
     include_package_data=True,
     install_requires=[
         "pytket ~= 1.18",
-        "qiskit ~= 0.43.1",
-        "qiskit-ibm-runtime ~= 0.11.1",
-        "qiskit-aer ~= 0.12.0",
-        "qiskit-ibm-provider ~= 0.6.1",
+        "qiskit ~= 0.44.0",
+        "qiskit-ibm-runtime ~= 0.11.3",
```

Comment:
Should this be listed in the changelog as well?

Reply:
Done.